cpp/lib/include/MarcRecord.h

```diff
@@ -0,0 +1,166 @@
+/* 
+ * File:   MarcRecord.h
+ * Author: quboo01
+ *
+ * Created on 12. September 2016, 11:28
+ */
+
+#ifndef MARCRECORD_H
```

Comment:
The define does not follow our pattern, also why aren't all three, MarcRecord, MarcReader and MarcWriter in MarcUtil?  They could then be named Record, Reader and Writer.